src/components/editor/markdown-preview/regex-plugins/replace-pdf-short-code.ts

```diff
@@ -0,0 +1,11 @@
+import { RegexOptions } from '../../../../external-types/markdown-it-regex/interface'
+
+export const replacePdfShortCode: RegexOptions = {
+  name: 'legacy-pdf-short-code',
```

Comment:
shold it really be called legacy if there is no new version?

Reply:
fixed